dae/dae/genomic_resources/repository.py

```diff
@@ -280,6 +270,19 @@ def __init__(
         self.proto = protocol
         self._manifest: Optional[Manifest] = manifest
 
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, GenomicResource):
+            return False
+        return self.resource_id == other.resource_id and \
+            self.version == other.version and \
+            self.config == other.config and \
+            self._manifest == other._manifest
+
+    def __hash__(self) -> int:
+        return hash(self.resource_id
+                    + ".".join(map(str, self.version))
+                    + self.proto.get_url())
+

```

Comment:
if __eq__ depends on the resource config, probably the hash should depend on config too